core/build.gradle

```diff
@@ -115,7 +115,7 @@ dependencies {
     implementation "io.netty:netty-transport-native-epoll:${managedVersions['io.netty:netty-transport-native-epoll']}:linux-x86_64"
     implementation 'io.netty:netty-tcnative-boringssl-static'
     implementation 'io.netty:netty-handler-proxy'
-    implementation "io.netty.incubator:netty-incubator-transport-native-io_uring:${managedVersions['io.netty.incubator:netty-incubator-transport-native-io_uring']}:linux-x86_64"
+    optionalImplementation "io.netty.incubator:netty-incubator-transport-native-io_uring:${managedVersions['io.netty.incubator:netty-incubator-transport-native-io_uring']}:linux-x86_64"
```

Comment:
Changed to opt-in because this feature is highly experimental.

Reply:
IIUC, no tests will be run with this on the classpath now. Does it make sense to have some small integration tests which include it on the classpath to make sure something doesn't completely break because of it?

Reply:
At least the core will run with it on the classpath, no?

Reply:
Ah I think so - seems fine